<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fenoptions</class>
 <widget class="QMainWindow" name="Fenoptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Boutonaffoptionvoc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boutonaffoptionvid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Vidéo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boutonaffoptionchat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Chat</string>
    </property>
   </widget>
   <widget class="QWidget" name="fenoptionchat" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Historique conversation :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bout2m">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>2 mois</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bout3j">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>3 jours</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bout2s">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>2 semaines</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="montrerqdecrit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Me montrer quand j'écris</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="affemoticones">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher les émoticones</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="notmsg">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Notifications messages</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Boutonaffgeneral">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string>Général</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="Fengeneral" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="absauboutde5min">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Absent à partir de 5 minutes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="modesilencieux">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode silencieux</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="lancerlqaudem">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Lancement de LoQuiî avec l'OS</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="codemlq">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection au demarrage de LoQuiî</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="activmaj">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Activer les mises à jour</string>
     </property>
    </widget>
    <widget class="QComboBox" name="choix_langue">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Français</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Anglais</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Espagnol</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Langue :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Boutonsupprnot2s">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>2 semaines</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Boutonsupprnot2m">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>2 mois</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Boutonsupprnot3j">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>3 jours</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Historique actualité :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ChoixCouleurNot">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Noir</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bleu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rouge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vert</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jaune</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Orange</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Violet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rose</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>220</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Couleur actualité :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fenoptionaudio" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QComboBox" name="ListePeriAudio">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mon périphérique audio :</string>
     </property>
    </widget>
    <widget class="QDial" name="ChoixVolumeAudio">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="ProgressVolumeAudio">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume :</string>
     </property>
    </widget>
    <widget class="QSlider" name="QuaAudio">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Qualité :</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckEnrAudio">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Fonctionnalité d'enregistrement audio</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fenoptionvideo" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QComboBox" name="ListePeriVideo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mon périphérique vidéo :</string>
     </property>
    </widget>
    <widget class="QSlider" name="QuaVideo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
     border: 1px solid #999999;
     height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
     margin: 2px 0;
 }
 
 QSlider::handle:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
     border: 1px solid #5c5c5c;
     width: 18px;
     margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
     border-radius: 3px;
 }</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Qualité :</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="ImgVideoOptions">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="CheckDemAutoVideo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Demarrer automatiquement ma vidéo</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
